src/_data/languages.json

```diff
@@ -8,7 +8,7 @@
         {
             "flag": "🇯🇵",
             "code": "jp",
-            "name": "Japenese – 日本語",
+            "name": "Japanese – 日本語",
```

Comment:
This file was actually removed. I think if you rebase onto main, this file will disappear.